modules/appeals_api/spec/requests/v2/supplemental_claims_controller_spec.rb

```diff
@@ -96,7 +96,7 @@ def new_base_path(path)
         sc = AppealsApi::SupplementalClaim.find(parsed['data']['id'])
         data_evidence_type = JSON.parse(data).dig(*%w[data attributes evidenceSubmission evidenceType])
 
-        expect(sc.metadata).to eq({ 'evidence_type' => data_evidence_type })
+        expect(sc.metadata['evidence_type']).to eq(data_evidence_type)
```

Comment:
Since evidence_type in no longer the only key-value pair in out metadata hash, i had to change this existing test to go after the specific evidence_type key-value.

I'm testing this new business line metadata down in the model spec, since that is where we grab the LOB from the form data and write it to the metadata column via an active record create hook